docs/api/telegram.rst

```diff
@@ -2077,6 +2078,38 @@ about its business.
       *This attribute can be None if it's not provided by Telegram.*
 
 
+.. py:class:: botogram.CallbackQuery
+
+      This class represents a callback query object.
+   .. py:attribute:: id
+
+      Unique identifier for the query
+
+   .. py:attribute:: sender
+
+      The :py:class:`~botogram.User` that sends the callback query
+
+   .. py:attribute:: message
+
+      :py:class:`~botogram.Message` with the callback button that originated the query.
```

Comment:
Don't add dots at the end only in some snippets, add them everywhere or nowhere.